src/spaceObjects/playerSpaceship.cpp

```diff
@@ -1117,9 +1117,9 @@ void PlayerSpaceship::setRepairCrewCount(int amount)
 
 void PlayerSpaceship::addToShipLog(string message, glm::u8vec4 color)
 {
-    // Cap the ship's log size to 100 entries. If it exceeds that limit,
+    // Cap the ship's log size to 10000 entries. If it exceeds that limit,
     // start erasing entries from the beginning.
-    if (ships_log.size() > 100)
+    if (ships_log.size() > 10000)
```

Comment:
Suggest reducing to 1,000, which is still a 10-fold increase, or less. Barring any changes to replication behavior, log rendering, or the GuiSlider behavior, this may be the practical limit without significantly degrading the log UI or performance. Naive testing suggests this still increases packet size each log update by ~20kB vs. 100 (vs. 200kB+ increase at 10000). Scenarios with noisier logs, like those with lots of CpuShips reporting order changes and status, might be more affected than others.

```suggestion
    if (ships_log.size() > 1000)
```